### PR TITLE
Products

### PR DESCRIPTION
**Pull request by chanchal16**
- made product listing page
- added filters.
- Preview link: https://priceless-bhaskara-f6c5a8.netlify.app/features/products.html

### CODE REVIEW

**Comment by sandeepkumardev-zz** [comment]
looks good to me

**Review by biswanath-tewari — APPROVED** [review]

File: features/products.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+                    <div class="card-content">
+                        <div class="content-title">
+                            <h4>Fastrack</h4>
+                            <span class=" gray">
+                                <i class="far fa-heart "></i>
+                            </span>
+                        </div>
+                        <div class="desc">
+                            <p class="desc-title">Eyeglasses</p>
+                            <p><strong> Rs.799 </strong> <span class="strike-text gray">Rs.999
```
Comment (rishbitsnbytes) [inline features/products.html:100]: ₹ would be better instead of Rs.

File: features/products.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+            </ul>
+        </nav>
+    </header>
+    <!----main section--->
+    <main>
+        <!---sidebar---->
+        <aside>
+            <div class="sidebar">
+                <span class="h6 sidebar-heading">Filters</span>
+                <div class="filters">
```
Comment (rishbitsnbytes) [inline features/products.html:34]: can be more semantic like section tag or something like that.

**Review by rishbitsnbytes — COMMENTED** [review]
Everything looks good to me except few minors.

**Pull request merged**